packages/ember-testing/lib/test.js

```diff
@@ -467,6 +469,107 @@ EmberApplication.reopen({
   }
 });
 
+if (isEnabled('ember-testing-instances')) {
+  EmberApplication.reopen({
+    /**
+      Builds a new instance of the application and registers test helpers on it.
+
+      Example:
+
+      ```javascript
+      var instance = run(App, 'buildTestInstance');
```

Comment:
i wonder if we should do a run.join inside of this method, so we don't need to enforce the external run.